src/admin/dumb_components/letter_body.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export default ({ letter }) => {
+
+  const formatContent = (content) => (
+    <div>
+      {content.split('\n').map((paragraph, i) =>
```

Comment:
I tend to avoid `map`s that take the index for a couple of reasons. One that's relevant here is that for React, the index in the array is frequently a pretty bad choice for the key (it's not a problem here).

Another is that passing in the index breaks the functor laws. Having your API follow laws is really helpful!

It's not a big deal and this solution is very clean